fastai/losses.py

```diff
@@ -51,6 +51,8 @@ class BCEWithLogitsLossFlat(BaseLoss):
     "Same as `nn.BCEWithLogitsLoss`, but flattens input and target."
     @use_kwargs_dict(keep=True, weight=None, reduction='mean', pos_weight=None)
     def __init__(self, *args, axis=-1, floatify=True, thresh=0.5, **kwargs):
+        # `flatten`=True throws a RuntimeError due to shape mismatch
+        if kwargs.get('pos_weight') is not None: kwargs['flatten'] = False
```

Comment:
```suggestion
        if kwargs.get('pos_weight', None) is not None: kwargs['flatten'] = False
```

Reply:
Added in 732a85cfcff77bcef458a4b32188e9915cad1b1a